loadbalancer/deploy

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-# Node because the secrets can change we may have to delete the stack to update
-docker stack deploy -c docker-compose.lb.yml lb
+# Note because the secrets can change we may have to delete the stack and recreate it to update
```

Comment:
This may have changed in recent light of docker swarm changes.

Reply:
I had to re-create the stack in order to update the Traefik config.

Though, I've now changed it to use "configs" instead of "secrets" which you may be right about - will have to investigate.